internal/pkg/term/prompt/select.go

```diff
@@ -21,6 +21,8 @@ const (
 	noAdditionalFormatting = 0
 )
 
+var regexpSGR = regexp.MustCompile("\x1b\\[([0-9]{1,2}(;[0-9]{1,2})?)?m")
```

Comment:
Can we include a comment here to explain what SGR stands for an example input that matches the regex 🙏 ? I can never read regexes 🤦